### PR TITLE
added generics basics sample

### PR DESCRIPTION
**Pull request by nirutadhital**

### CODE REVIEW

File: GenericsBasics.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace GenericsBasics {
+    public class Vessel<T> {
+        protected T Capacity;
+
+        public Vessel(T capacity) {
+            this.Capacity = capacity;
```
Comment (nirutadhital) [inline GenericsBasics.cs]: Change this field to lowercase or camelCase

File: GenericsBasics.cs
Comment (Puja-knegi) [inline GenericsBasics.cs:1]: rrtert

**Review by Puja-knegi — CHANGES_REQUESTED** [review]
change the name

**Review by Puja-knegi — APPROVED** [review]
good

**Pull request merged**